sinks/splunk/splunk.go

```diff
@@ -18,114 +18,260 @@ import (
 	"github.com/stripe/veneur/trace/metrics"
 )
 
+// TestableSplunkSpanSink provides methods that are useful for testing
+// a splunk span sink.
+type TestableSplunkSpanSink interface {
```

Comment:
Does this need to be exported? Is the idea that callers would use it too?

Reply:
It is used only in the splunk_test.go file, which is using the external sink interface only. I can make the test be internal, but it seems pleasing to have only that one dependency. I could go either way, your call.

Reply:
Ah, I missed that this was a whitebox test!